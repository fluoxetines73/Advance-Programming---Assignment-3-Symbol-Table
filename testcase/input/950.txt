RPRINT
INSERT x string
ASSIGN x 'HelloWorld' random
LOOKUP z
LOOKUP x
RPRINT
BEGIN
RPRINT
LOOKUP x
ASSIGN x '1234ABC'
RPRINT
INSERT e_123_2r string
LOOKUP e_123_2r
UWAAAA
RPRINT
BEGIN
INSERT e_123_2r number random
INSERT y string
UWAAAA
END
END