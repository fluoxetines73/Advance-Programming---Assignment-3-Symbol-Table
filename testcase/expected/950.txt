Invalid: ASSIGN x 'HelloWorld' random